### ISSUE TITLE
Language server errors whilst writing `COPY --from` line

### ISSUE THREAD

**Issue by henrahmagix** [microsoft/vscode-docker#3349]
This means VS Code alerts me with a popup in the bottom right, saying:
>A request has failed. See the output for more information.

Viewing the output for "Dockerfile Language Server", I see multiple repetitions of the following:
```
[Error - 13:44:19] Request textDocument/completion failed.
  Message: Request textDocument/completion failed with message: Cannot read property 'getRange' of undefined
  Code: -32603 
```

To reproduce, in a Dockerfile, start writing `COPY --from=something`: the line will be highlighted red and the tooltip info will say "COPY requires at least two arguments - dockerfile-utils(7)" – this is helpful =) Then start typing a space to start writing your two arguments, and the error above occurs immediately after typing the space. If you continue to write out the two arguments, no further error occurs; but if you go back to the `--from` argument and retype the space, then the error occurs every time you do that.

- No error except tooltip highlight: `COPY --from=something`
- Error occurs: `COPY --from=something ` (I typed a non-breaking space here because the highlight wasn't showing the extra space: don't copy this)
- No error except tooltip highlight: `COPY --from=something /path/to/something`
- All good: `COPY --from=something /path/to/something ./something`

**Comment by rcjsuen** [comment]
@henrahmagix Thank you for the bug report. I was able to successfully reproduce this and will take a look at it.

**Comment by rcjsuen** [comment]
I've opened https://github.com/rcjsuen/dockerfile-language-server-nodejs/issues/258 to track this.

**Comment by bwateratmsft** [comment]
Thanks @rcjsuen! Like #3327, I'll close this one since language server updates are regularly picked up.

**Comment by henrahmagix** [comment]
Thanks so much!

**Comment by bwateratmsft** [comment]
This is now released in Docker extension version 1.19.0.
